src/nlr/Layer.cpp

```diff
@@ -157,6 +158,18 @@ void Layer::computeAssignment()
         }
     }
 
+    else if ( _type == LEAKY_RELU )
+    {
+        double slope = _alpha;
+        for ( unsigned i = 0; i < _size; ++i )
+        {
+            NeuronIndex sourceIndex = *_neuronToActivationSources[i].begin();
+            double inputValue = _layerOwner->getLayer( sourceIndex._layer )->getAssignment( sourceIndex._neuron );
+
+            _assignment[i] = FloatUtils::max( inputValue, slope * inputValue );
```

Comment:
unclear. is slope always <=1?
even if so, lets make this more comprehensible

Reply:
Yes, currently we need slope to be <= 1, otherwise the symbolic bounds would be incorrect. I added an assertion and also throw an error when this requirement is not met when constructing a LeakyReLU